build_image

```diff
@@ -146,8 +143,6 @@ fi
 mkdir -p "${BUILD_DIR}"
 
 DISK_LAYOUT="${FLAGS_disk_layout:-base}"
-DEV_DISK_LAYOUT="${FLAGS_disk_layout:-devel}"
```

Comment:
I think this is the only place the `devel` layout is used, so that could be dropped as well.

Reply:
Done (and verified the dev container and prod image still build).